gradle/dependencies.gradle

```diff
@@ -13,16 +13,16 @@ ext {
   // version which we should use in this build
   def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
   if (mapboxNavigatorVersion == null || mapboxNavigatorVersion == '') {
-      mapboxNavigatorVersion = '119.0.1'
+      mapboxNavigatorVersion = '121.0.0'
   }
   println("Navigation Native version: " + mapboxNavigatorVersion)
 
   version = [
-      mapboxMapSdk              : '10.9.0',
+      mapboxMapSdk              : '10.10.0-beta.1',
       mapboxSdkServices         : '6.10.0-beta.1',
       mapboxCore                : '5.0.2',
       mapboxNavigator           : "${mapboxNavigatorVersion}",
-      mapboxCommonNative        : '23.1.1',
+      mapboxCommonNative        : '23.2.0-beta.1',
       mapboxSearchSdk           : '1.0.0-beta.39',
```

Comment:
Search SDK uses the old Common SDK version. Can you check how search works in Android Auto?
This will become important when we integrate this version in 1TAP next week.

Reply:
It won't be compatible unfortunately, refs https://mapbox.slack.com/archives/C039TRJJQ3A/p1668002405914209. We might need to follow up separately for Android Auto (and work with search team).

Reply:
hm, I've checked AA and search works

Reply:
Ah in case of search not integrating the breaking metrics function in Common we should be fine. The breakage definitely impacted maps but we're updating that as well 👍